### DISCUSSION TITLE
Update Umbrel immich

### DISCUSSION THREAD

**Discussion by 0ceanSlim** [immich-app/immich#3961] [General]
If the server and app are out of sync in version then the app can have issues. The Umbrel app needs to be updated every time the app is. The Umbrel version is almost always behind. If someone can walk me through the prs necessary to update, I'd be glad to maintain this myself. This is becoming unreliable on Umbrel.

**Comment by bo0tzz** [comment]
We don't manage anything with regards to Umbrel, that's all theirs. Looks like there is already a PR to update: https://github.com/getumbrel/umbrel-apps/pull/761
